source/ota.c

```diff
@@ -324,6 +326,8 @@ static const char * pOtaAgentStateStrings[ OtaAgentStateAll + 1 ] =
     "All"
 };
 
+
+
```

Comment:
Nit: These extra lines can be removed

Reply:
Done: https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/112/commits/130a2cafd84f8880cf198de5ac2fe2542fdcd831